azkaban-common/src/main/java/azkaban/executor/Status.java

```diff
@@ -30,6 +30,8 @@
   SUCCEEDED(50),
   KILLING(55),
   KILLED(60),
+  // EXECUTION_STOPPED refers to a terminal flow status due to crashed executor/container
+  EXECUTION_STOPPED(65),
```

Comment:
Could the name of the status be confusing to users? It says the execution stopped but is it a temporary stop? Will it continue to run at some point? To me the name does not suggest a final status but a transition one. WDYT @sshardool @jakhani @djaiswal83 @aditya1105